scripts-build

```diff
@@ -4,11 +4,8 @@ set -e
 set -u
 set -x
 
-export TARGET=${TARGET:-..}
-export OMERO_DIST=${OMERO_DIST:-/opt/omero/server/OMERO.server}
-export PYTHONPATH=${OMERO_DIST}/lib/python
 
 cd $TARGET
 
-pip install --upgrade pip setuptools
+#pip install --upgrade pip setuptools
```

Comment:
Did you intend to remove this?

Reply:
yes
let me remove the comment.
I had to move it to another file, since i now use ``--user`` to run the tests (failures in ``ome/scripts`` otherwise)